[docs-only] Add a few words about idm and how to enable it (#3372)
owncloud/ocis: docs/extensions/idm/setup.md

---
title: Service Setup
date: 2022-03-22T00:00:00+00:00
weight: 20
geekdocRepo: https://github.com/owncloud/ocis
geekdocEditPath: edit/master/docs/extensions/idm
geekdocFilePath: setup.md
geekdocCollapseSection: true
---

{{< toc >}}

## Using ocis with libregraph/idm

> Currently ocis will still run the accounts and glauth services to manage users. Until the default is switched

Reviewer: ```suggestion
Currently, oCIS still runs the accounts and glauth services to manage users. Until the default is switched
```

to libregraph/idm, ocis has to be started with a custom configuration in order to use libregraph/idm as

> the users and groups backend (this setup also disables the glauth and accounts serice:

Reviewer 2: ```suggestion
the users and groups backend (this setup also disables the glauth and accounts service):
```

Author: fixed

> the users and groups backend (this setup also disables the glauth and accounts service:

Reviewer: ```suggestion
the users and groups backend (this setup also disables the glauth and accounts service):
```
